tox.ini

```diff
@@ -15,6 +14,11 @@ deps =
     pytest-timeout
     pytest-openfiles
     pytest-flakefinder
+
+[testenv:py{27,35,36,37,38,39,310,launch}]
```

Comment:
While you're at it...maybe ditch the 27 and 35 environments for good from `tox.ini`?